images/mod-tile/README.md

```diff
@@ -29,6 +29,13 @@ The sequence number of last rendered replication will be stated in `/mnt/expired
 - `MOD_TILE_PATH` e.g `hot`
 - `REPLICATION_DIR` the inner directory path for mounted replications
 - `EXPIRED_DIR` the inner directory path for the expired lists
+- `LOG_CONFIG_PATH` logging configuration yaml file path
```

Comment:
LOG_CONFIG_PATH is not used by the script.

Reply:
done